tests/test_fields.py

```diff
@@ -56,7 +59,10 @@ def test_text_good(value, kwargs):
     ('x', dict(max_length=80, min_length=2), rules.cell.MinChar),
 ])
 def test_text_bad(value, kwargs, rule):
-    assert field.Text(**kwargs)._has_error(value, rule.rule_exception())
+    fld = field.Text(**kwargs)
+    assert fld._has_error(value, rule.rule_exception())
+    with pytest.raises(AssertionError):
```

Comment:
There is a bit of a gap here, where the harness could throw an AssertionError, but only for one of the file types. In the case where we're expecting all files to have an error, this could leave a gap in the testing. This is not a deal breaker, but something to think on for how we might improve the harness in the future.

Reply:
I've completed the other pieces of feedback, but I kinda want to take a crack at this. But perhaps we should release what we have so that our users can take advantage of the Excel fix, and I can attempt to rework some of this file harness in a separate 'feature' branch. Does that sound like a plan?